### DISCUSSION TITLE
Error when creating Cascade Classifier?

### DISCUSSION THREAD

**Discussion by ScottDurkin** [emgucv/emgucv#540] [Q&A]
Hi all,

I am looking into EmguCV for the first time ever and  when I try to create a Cascade, I get the following error message on runtime..
![image](https://user-images.githubusercontent.com/24443043/124837033-00f61c80-df7c-11eb-9b1c-a894ae3c1ed6.png)

When I add the cvextern to the project, I get the following error.
![image](https://user-images.githubusercontent.com/24443043/124837071-1408ec80-df7c-11eb-8f19-0460f5ad1b1d.png)

Does anyone know how I can resolve this? I have the xml file and cvextern dll are within the project and "copy to output directory" is set to "copy always"

Thank you in advance.

**Comment by emgucv** [comment]
Please follow [this instruction] (https://www.emgu.com/wiki/index.php/Download_And_Installation#The_type_initializer_for_.27Emgu.CV.CvInvoke.27_threw_an_exception.) to trouble shot.
